pkg/rid/store/cockroach/store.go

```diff
@@ -46,6 +51,38 @@ type Store struct {
 	clock  clockwork.Clock
 }
 
+// NewStore returns a Store instance connected to a cockroach instance via db.
+func NewStore(ctx context.Context, db *cockroach.DB, logger *zap.Logger) (*Store, error) {
+	store := &Store{
+		db:     db,
+		logger: logger,
+		clock:  DefaultClock,
+	}
+
+	if err := store.CheckCurrentMajorSchemaVersion(ctx); err != nil {
+		return nil, stacktrace.Propagate(err, "Failed to construct store instance for remote ID")
+	}
+
+	return store, nil
+}
+
+// CheckCurrentMajorSchemaVersion checks that store supports the current major schema version.
+func (s *Store) CheckCurrentMajorSchemaVersion(ctx context.Context) error {
+	vs, err := s.GetVersion(ctx)
+	if err != nil {
+		return stacktrace.Propagate(err, "Failed to get database schema version for remote ID")
+	}
+	if vs == cockroach.UnknownVersion {
+		return stacktrace.NewError("Remote ID database has not been bootstrapped with Schema Manager, Please check https://github.com/interuss/dss/tree/master/build#updgrading-database-schemas")
+	}
+
+	if currentMajorSchemaVersion != vs.Major {
+		return stacktrace.NewError("Unsupported schema version for remote ID! Got %s, requires major version of %d.", vs, currentMajorSchemaVersion)
```

Comment:
nit: This is likely to be an error observed directly by DSS operators; can we elaborate on how to fix it with a link to the instructions like line 76?